runtime/hydradx/src/tests.rs

```diff
@@ -363,3 +367,24 @@ mod xcm_fee_payment_api_tests {
 		});
 	}
 }
+
+#[test]
+fn assert_kusama_root_account() {
+	// Initialize the Externalities environment
+	sp_io::TestExternalities::new_empty().execute_with(|| {
+		let ksm_root_location = Location::new(2, [GlobalConsensus(Kusama)]);
+		let ksm_root_account =
+			GlobalConsensusConvertsFor::<UniversalLocation, AccountId>::convert_location(&ksm_root_location)
+				.expect("Failed to convert location");
+
+		// // Example treasury account, replace with the actual expected value
```

Comment:
comment feels a bit redundant, I would just name the variable to `expected_treasury_account` and remove the comment.